web/post-restoration-transition-application/nuxt.config.ts

```diff
@@ -115,10 +113,6 @@ export default defineNuxtConfig({
       version: `Post Restoration Transition Application UI v${process.env.npm_package_version || ''}`,
       preexistingCompanyProvisions: process.env.PREEXISTING_COMPANY_PROVISIONS_URL
         || 'https://www.corporateonline.gov.bc.ca/WebHelp/provisions.htm',
-      addressCompleteKey: process.env.NUXT_ADDRESS_COMPLETE_KEY,
-      legalApiUrl: `${process.env.NUXT_LEGAL_API_URL}${process.env.NUXT_LEGAL_API_VERSION}`,
-      businessDashboardUrl: process.env.NUXT_BUSINESS_DASHBOARD_URL,
-      registryHomeUrl: process.env.NUXT_REGISTRY_HOME_URL,
       brdUrl: process.env.NUXT_BUSINESS_REGISTRY_DASHBOARD_URL,
```

Comment:
can you remove this and update the reference in the code to `businessRegistryDashboardUrl` please? The less manually mapped things in here the better